oceanwaters/doc/setup_oceanwaters.md

```diff
@@ -1,61 +1,103 @@
 You can perform operations like update, diff, and status on multiple
 repositories at once.
 
-Create OceanWATERS Workspace
-----------------------------
-* Create your workspace directory
+To create a ROS workspace for OceanWATERS first use the setup.bash script
+provided in the ROS installation to set required ROS environment variables:
+
+```
+source /opt/ros/melodic/setup.bash
+```
+
+For convenience users may want to add the above command to their ~/.bashrc
+file.
+
+Next create a workspace directory for OceanWATERS with a src subdirectory, and
+go to the src subdirectory:
+
 ```
 mkdir -p oceanwaters_ws/src
 cd oceanwaters_ws/src
 ```
-* Next, check out the top level oceanwaters repo which contains the default
-.rosinstall workspaces
+
+Then checkout the top level OceanWATERS git repository:
+
 ```
 git clone https://github.com/nasa/ow_simulator.git
 ```
-* Symlink the .rosinstall you're interested in into the src directory and run
-wstool. This will check out all the repos necessary to build the workspace.
+
+ROS’ wstool will be used to checkout additional repositories containing the
+source code files making up OceanWATERS. The wstool utility will look for a
+file, `oceanwaters_ws/src/.rosinstall`, that contains a list of repositories to
+check out or update. The OceanWATERS repository contains a number of `.rosinstall`
+files for different uses. Create a link to the default repository list:
+
 ```
 ln -s ow_simulator/oceanwaters/workspaces/oceanwaters.rosinstall .rosinstall
-wstool up
 ```
-* To build the workspace, cd back up to the workspace directory, source the ROS
-environment, and run catkin build. The first time you build should take an extra
-long time as the build process downloads around 2 GB of 3rd party code and data.
-The code and data will be cached in the src/ tree, so subsequent builds should
-be much faster.
+
+Now use wstool to checkout/update the source code:
+
 ```
-cd ..
-source /opt/ros/melodic/setup.bash
-catkin build
+wstool update
 ```
-* NOTE: if catkin is not found, it was somehow left out of your ROS install.  Try:
+
+OceanWATERS Build
+-----------------
+
+To build the workspace, cd back up to the workspace directory and run catkin build.
+
 ```
-sudo apt-get install python-catkin-tools
+cd ..
+catkin build
 ```
-* The workspace should now be compiled and ready to run. Source the workspace
-environment.
+
+The first build of OceanWATERS will take an extra long time as the build process downloads
+around 2 GB of 3rd party code and data. The code and data will be cached in the src/ tree,
+so subsequent builds should be much faster.
+
+Finally set OceanWATERS specific environment variables using the provided
+script:
+
 ```
-source devel/setup.bash
+source oceanwaters_ws/devel/setup.bash
 ```
-* Launch the sim
+
+At this point you should see Gazebo related variables such as
+`GAZEBO_MODEL_PATH` set in your environment.  You may wish to add the line above to your
+.bashrc file.
```

Comment:
Sorry, one more edit. I think sourcing setup.bash in your .bashrc is risky for most users. This would only be useful if you only use one ROS workspace and don't plan to recompile it much. I think this suggestion should be removed.